File: src/main/java/com/secacon/secasignbox/sdk/dto/sign/ReadOrganizationDocumentDto.java
package com.secacon.secasignbox.sdk.dto.sign;

import com.secacon.secasignbox.sdk.dto.document.DocumentSigningStatusDto;
import com.secacon.secasignbox.sdk.dto.document.ReadDocumentDto;
import com.secacon.secasignbox.sdk.dto.document.SignatureTypeDto;

public record ReadOrganizationDocumentDto(

    DocumentSigningStatusDto signingStatus,

    String errorMessage,

    String documentName,

    String encodedSignedDocumentOrSignature,

    SignatureTypeDto signatureType,

    ReadDocumentDto document
) {
}

File: csharp/Secasign/dto/sign/ReadOrganizationDocumentDto.cs
using Secasign.dto.document;

namespace Secasign.dto.sign;

public record ReadOrganizationDocumentDto(DocumentSigningStatusDto? signingStatus, string? errorMessage, string documentName, string? encodedSignedDocumentOrSignature, SignatureTypeDto? signatureType, ReadDocumentDto? document);
